index.html

```diff
@@ -365,9 +365,9 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 <dl data-link-for="PerformanceResourceTiming">
 <dt><dfn>name</dfn></dt>
 <dd>This attribute MUST return the <a data-cite=
-"HTML#resolve-a-url">resolved URL</a> of the requested resource.
-This attribute MUST NOT change even if the <a data-cite=
-"FETCH#concept-fetch">fetch</a> redirected to a different URL.</dd>
+"FETCH#concept-request-url">URL</a> of the resource's <a data-cite=
```

Comment:
Looking at https://fetch.spec.whatwg.org/#concept-request-url I don't see it being explicitly set to the first URL in the list, and there's a not saying "implementations are encouraged" to do that. As such, I don't know if this is something we want to rely on. Maybe we should explicitly point to the first URL in the [URLList](https://fetch.spec.whatwg.org/#concept-request-url-list)?

/cc @annevk for his opinions

Reply:
Seems like `URL` is set to be exactly that, judging by the fact that during redirects it's not changed.
But I would let @annevk confirm.

Reply:
Yeah, looking closer it seems to be set in the Request constructor and not modified on redirects.